components/brave_wallet/browser/eth_tx_controller_unittest.cc

```diff
@@ -76,10 +76,13 @@ class TestEthTxControllerObserver
     tx_updated_ = true;
   }
 
-  void OnTransactionStatusChanged(mojom::TransactionInfoPtr tx) override {}
+  void OnTransactionStatusChanged(mojom::TransactionInfoPtr tx) override {
+    tx_changed_called_ = true;
+  }
 
   bool TxUpdated() { return tx_updated_; }
-
+  bool TxObserverChanged() { return tx_changed_called_; }
+  void ResetObserverChanges() { tx_changed_called_ = false; }
```

Comment:
Rename to `Reset()` and reset tx_updated_  to false too in this function.

Reply:
done